config/crds/patches/core.kubeadmiral.io_propagationpolicies.sh

```diff
@@ -0,0 +1,3 @@
+# controller-gen does not respect {} as default value for a struct field
+# issue: https://github.com/kubernetes-sigs/controller-tools/issues/622
```

Comment:
Can we move this comment (and the identical comment in CPP patch) into the generic patch?